### PR TITLE
implemented new style for tasks and buttons to move them between tabs

### PR DESCRIPTION
**Pull request by CptLala**

### CODE REVIEW

File: src/shared/ListItem/ListItem.tsx [hunk 1]
```diff
@@ -56,11 +56,11 @@ const ListItem = ({ common, onClickFunc, editFunc, deleteFunc }: PropTypes) => {
 
   return (
     <Swipeable rightButtons={buttons.length > 0 ? buttons : null}>
-      <View style={styles.item}>
-        <TouchableWithoutFeedback onPress={handleOnClick}>
+      <TouchableWithoutFeedback onPress={handleOnClick}>
```
Comment (lukasbals) [inline src/shared/ListItem/ListItem.tsx]: When I tried that I had the problem that the "onPress" hook was calld even if I just wanted to slide the item. Is that solved somehow or is it a different behaviour on an other device?

**Review by lukasbals — COMMENTED** [review]
Otherwise it looks good to me!

File: src/features/TaskScreen/TaskScreen.tsx [hunk 2]
```diff
@@ -86,29 +86,34 @@ function TaskScreen({ route, navigation }: TaskScreenPropTypes) {
...
+        );
+      case State.InProgress.toString():
         return (
           <TaskListView
             tasks={tasks.filter(
-              (task: Task) => task.state === Status.InProgress
+              (task: Task) => task.state === State.InProgress
             )}
-            addFunc={addTaskFunc}
+            addFunc={null}
```
Comment (BakaBoing) [inline src/features/TaskScreen/TaskScreen.tsx:112]: Why not enabling addFunc like in ToDo?

File: src/features/TaskScreen/TaskScreen.tsx [hunk 3]
```diff
@@ -86,29 +86,34 @@ function TaskScreen({ route, navigation }: TaskScreenPropTypes) {
...
           />
         );
-      case Status.Verify.toString():
+      case State.Verify.toString():
         return (
           <TaskListView
-            tasks={tasks.filter((task: Task) => task.state === Status.Verify)}
-            addFunc={addTaskFunc}
+            tasks={tasks.filter((task: Task) => task.state === State.Verify)}
+            addFunc={null}
```
Comment (BakaBoing) [inline src/features/TaskScreen/TaskScreen.tsx:123]: Why not enabling addFunc like in ToDo?

File: src/features/TaskScreen/TaskScreen.tsx [hunk 4]
```diff
@@ -86,29 +86,34 @@ function TaskScreen({ route, navigation }: TaskScreenPropTypes) {
...
           />
         );
-      case Status.Done.toString():
+      case State.Done.toString():
         return (
           <TaskListView
-            tasks={tasks.filter((task: Task) => task.state === Status.Done)}
-            addFunc={addTaskFunc}
+            tasks={tasks.filter((task: Task) => task.state === State.Done)}
+            addFunc={null}
```
Comment (BakaBoing) [inline src/features/TaskScreen/TaskScreen.tsx:134]: Why not enabling addFunc like in ToDo?

File: src/shared/ListItem/ListItem.tsx (hunk 1 above)
Comment (BakaBoing) [reply src/shared/ListItem/ListItem.tsx:59]: on my Android, the onPress is not called, check on your phone please

File: src/features/TaskScreen/TaskScreen.tsx (hunk 2 above)
Comment (lukasbals) [reply src/features/TaskScreen/TaskScreen.tsx:112]: I think we wanted to only be able to create tasks in the ToDo column like it is implemented in the browser version of the app. 🤔 But we can discuss this again.

Comment (BakaBoing) [reply src/features/TaskScreen/TaskScreen.tsx:112]: I think I missed that xD

**Review by lukasbals — APPROVED** [review]

**Pull request merged**